src/devices/Mfrc522/Mfrc522Controller.cs

```diff
@@ -0,0 +1,447 @@
+using System;
+using System.Diagnostics;
+using System.Device.Gpio;
+using System.Device.Spi;
+using System.Collections.Generic;
+
+using static Iot.Device.Mfrc522.Status;
+using static Iot.Device.Mfrc522.Register;
+using static Iot.Device.Mfrc522.RequestMode;
+using static Iot.Device.Mfrc522.Command;
+
+namespace Iot.Device.Mfrc522
+{
+    public class Mfrc522Controller : IDisposable
+    {
+        const byte NRSTPD = 22;
```

Comment:
this needs rename - also any reason this is not configurable? I presume this is pin for hard reset?

Reply:
Not got around doing that yet.